build-logic/integration-testing/src/main/kotlin/gradlebuild/integrationtests/shared-configuration.kt

```diff
@@ -276,3 +280,181 @@ fun Project.resolver(name: String, libraryElements: String, extends: Configurati
         extendsFrom(extends)
     }
 }
+
+private val ignoreWithIsolatedProjectsExecuter: List<String> = listOf(
```

Comment:
💅 Explicit type is unnecessary (member is private and it's initialized immediately from a list of strings)